tmol/optimization/modules.py

```diff
@@ -82,23 +66,29 @@ def __init__(self, score_system, dofs, kinforest, system_size, dof_mask=None):
         self.score_system = score_system
         self.kinforest = kinforest
 
+        if dof_mask is None:
+            dof_mask = torch.full(
+                dofs.shape[:-1], True, device=dofs.device, dtype=torch.bool
+            )
+
+        self.system_size = system_size
+
+        if dof_mask is None:
+            dof_mask = torch.full(
```

Comment:
This is unreachable / duplicated. I think it's safe to remove.

Reply:
will do